README.md

```diff
@@ -1,5 +1,10 @@
 # QCVIS — Interactive Visualization of Quantum Algorithms
 
+[![License](https://img.shields.io/github/license/fh-igd-iet/qcvis?color=blue&label=License)](https://fh-igd-iet.github.io/qcvis/)
+[![Try it out](https://img.shields.io/badge/-Try%20it%20out%20→-brightgreen "Open application")](https://fh-igd-iet.github.io/qcvis/)
+
+[![QCVIS in action](.github/assets/readme_screenshot.png "QCVIS in action")](https://fh-igd-iet.github.io/qcvis/)
```

Comment:
- I would remove the title (see previous comment) as it is identical with the alt-text
- Please use the following (admittedly edited for compactness) screenshot instead
  ![state-cube-animating-edit](https://user-images.githubusercontent.com/54402514/215765856-03a7b878-b20b-4932-8083-64d81b3db71b.png)

Reply:
Ok, accepted